ivy/functional/ivy/experimental/activations.py

```diff
@@ -232,15 +258,23 @@ def relu6(
     return current_backend(x).relu6(x, out=out)
 
 
+relu6.jax_like = _relu6_jax_like
+
+
 @handle_exceptions
 @handle_backend_invalid
 @handle_nestable
 @handle_array_like_without_promotion
 @handle_out_argument
 @to_native_arrays_and_back
 @handle_device_shifting
+@handle_complex_input
 def logsigmoid(
```

Comment:
Complex numbers are marked as unsupported for `logsigmoid` in the `tf` backend. Either a custom implementation should be added to that backend, or a comment along the lines of `# TODO: implement for complex numbers` should be added to it

Reply:
I think there's been a mix-up because `tf` doesn't have unsupported for complex

Reply:
as rega

> I think there's been a mix-up because `tf` doesn't have unsupported for complex

as regards the `logsigmoid`

Reply:
The `tf` backend uses `with_supported_dtypes` (rather than `with_*un*supported_dtypes`) for `logsigmoid`, which means only `float` types are supported at the moment (and everything else is unsupported).

Reply:
Thanks for pointing that out. I have now created the custom implementation for `logsigmoid` `tf`